c2corg_ui/static/partials/contexthelp/route-equipement_rating.html

```diff
@@ -0,0 +1,28 @@
+<p><strong translate>Protection</strong></p>
+<p translate>This fields details the presence and quality of fixed anchors (bolts, pitons, spits) as protection points for rock or ice climbing.</p>
+
+<p><strong translate>P1 well bolted :</strong></p>
+<ul>
+  <li translate>Sport climbing bolted with 10 or 12mm expansion bolts, chemical bolts, etc. The route may not be equiped for abseil.</li>
+  <li translate>Required gear : <span translate>12 to 14 clippers, some of them lengthy, 1 to 2 slings to complete equipment in grade II pitches.</span></li>
```

Comment:
In addition I see that a similar string is used in another partial => make sure the strings are exactly the same in order to have it only once in Transifex.